packages/cli/src/index.ts

```diff
@@ -14,6 +25,26 @@ program
     initCommand();
   });
 
+program
+  .command('sync')
+  .option('-a, --api-url <apiUrl>', 'The Novu Cloud API url, defaults to https://api.novu.co', 'https://api.novu.co')
+  .requiredOption(
+    '-b, --bridge-url <bridgeUrl>',
+    'The novu endpoint URL hosted in the bridge application, by convention ends in /api/novu'
+  )
```

Comment:
Small changes for consistency of casing.
```suggestion
  .requiredOption(
    '-b, --bridge-url <bridgeUrl>',
    'The Novu endpoint URL hosted in the Bridge application, by convention ends in /api/novu'
  )
```